lib/publiccloud/instance.pm

```diff
@@ -976,4 +976,40 @@ sub wait_for_state {
     die("The instance state is not '$state' but '$current' instead.");
 }
 
+sub check_selinux_pc {
```

Comment:
```suggestion
sub check_selinux_enforced {
```

The subroutine is under `lib/publiccloud/` so the `_pc` suffix is not needed.
The `check_selinux` is not very descriptive so adding `_enforced` tells you that it check if SELinux is in enforced mode.